index.js

```diff
@@ -31,16 +32,18 @@ module.exports = function (archive, target, opts, cb) {
   var entries = {}
   var watcher
 
-  if (opts.live) {
+  if (opts.watch && archive.live) {
     watcher = chokidar.watch([target], {
       persistent: true,
       ignored: opts.ignore
     })
     watcher.once('ready', function () {
       watcher.on('add', function (file, stat) {
+        status.emit('file watched', {path: file, mode: 'created'})
```

Comment:
Yea I wasn't sure what to call it, agree `file watched` is misleading. `file start watching` sounds to me like thats when we start watching it (not that it had changed).  Maybe `file watch event`?